src/game/multiboot/connection.c

```diff
@@ -177,8 +177,8 @@ void StartSinglePakConnect(void)
     }
 
     connectScreen->mbProgStart = gUnknown_080E01E0[connectScreen->unkFA][0];
-    connectScreen->mbProgLength = (u32)gUnknown_080E01E0[connectScreen->unkFA][1]
-        - (u32)gUnknown_080E01E0[connectScreen->unkFA][0];
+    connectScreen->mbProgLength = (uintptr_t)gUnknown_080E01E0[connectScreen->unkFA][1]
+        - (uintptr_t)gUnknown_080E01E0[connectScreen->unkFA][0];
```

Comment:
Length is a u32, I don't think it makes sense for this to be a pointer.

Reply:
Also in line 171 it uses 0xFC instead of the struct's size. Would be cool if you could fix that as well!

Reply:
This might be a mistake

Reply:
The length is u32, but that's fine because the distance between the pointers should always be u32